utility/service/service.go

```diff
@@ -147,3 +151,151 @@ func (r *relay) GetRelayChain() RelayChain              { return nil }
 func (r *relay) GetGeoZone() GeoZone                    { return nil }
 func (r *relay) GetToken() AAT                          { return nil }
 func (r *relay) GetSignature() string                   { return "" }
+
+type Provider interface {
```

Comment:
Rather than using a provider, use our `application-specific bus` which is what we use for dependency injection throughout the entire repository in every module/submodule.

You can then do `bus.GetConsensusModule().CurrentHeight()` or `bus.GetUtilityModule().GetSession(...)`.

For example, here is a mock of the utility module in `consensus/e2e_tests`:

![Screenshot 2023-05-25 at 2 47 27 PM](https://github.com/pokt-network/pocket/assets/1892194/26bcf1b5-4f0e-41da-876f-35c7eb596dca)

See this link for more details and please (please please) update the documentation where you see gaps: https://github.com/pokt-network/pocket/wiki/Modules-Readme

Reply:
Thank you for the review. Fixed.